docs/tutorial.md

```diff
@@ -3,7 +3,7 @@ id: tutorial
 title: Learn the Basics
 ---
 
-React Native is like React, but it uses native components instead of web components as building blocks. So to understand the basic structure of a React Native app, you need to understand some of the basic React concepts, like JSX, components, `state`, and `props`. If you already know React, you still need to learn some React-Native-specific stuff, like the native components. This tutorial is aimed at all audiences, whether you have React experience or not.
+React Native is like React, but it uses native components instead of web components as building blocks. So to understand the basic structure of a React Native app, you need to understand some of the basic React concepts, like JSX, components, `state`, and `props`. If you already know React, you still need to learn some specific React-Native stuff, like the native components. This tutorial is aimed at all audiences, whether you have React experience or not.
```

Comment:
```suggestion
React Native is like React, but it uses native components instead of web components as building blocks. So to understand the basic structure of a React Native app, you need to understand some of the basic React concepts, like JSX, components, `state`, and `props`. If you already know React, you still need to learn some React-Native specific stuff, like the native components. This tutorial is aimed at all audiences, whether you have React experience or not.
```